src/BookNavigator/search/search-results.js

```diff
@@ -90,10 +91,7 @@ export class IABookSearchResults extends LitElement {
   }
 
   dispatchSearchCanceled() {
-    this.dispatchEvent(new CustomEvent('bookSearchCanceled', {
-      bubbles: true,
-      composed: true,
-    }));
+    this.dispatchEvent(new CustomEvent('bookSearchCanceled'));
```

Comment:
Not a blocker, but for events that don't have any `details`, you can just use `new Event('foo')` instead of `CustomEvent`

Reply:
nice! TIL. will update